webpack/test-utils/react-testing-lib-wrapper.js

```diff
@@ -22,26 +23,29 @@ process.env.DEBUG_PRINT_LIMIT = 999999;
 function renderWithRedux(
   component,
   {
-    namespace, // redux namespace
-    initialState = { response: {}, status: STATUS.RESOLVED },
+    apiNamespace, // namespace if using API middleware
```

Comment:
It actually doesn't seem to be affected, but for consistency can you update [here](https://github.com/Katello/katello/blob/master/webpack/scenes/ContentViews/__tests__/contentViewPage.test.js#L18) to `apiNamespace`?